### DISCUSSION TITLE
Βαθμολογία εργασίας εξεταστική Φεβρουαρίου

### DISCUSSION THREAD

**Discussion by p17balo** [courses-ionio/hci#1760] [Announcements]
Καλησπέρα κύριε καθηγητά , κοίταξα την κατάταξη με τις βαθμολογίες και το ΑΜ μου δεν υπάρχει μέσα σε αυτήν , έχει γίνει κάποιο λάθος ; Επειδή έχω κάνει pull request

**Comment by epidrome** [comment]
https://github.com/courses-ionio/hci/discussions/1758#discussioncomment-2222956
